common-files/constants/sdk.mjs

```diff
@@ -4,13 +4,6 @@ export const TOKEN_TYPE = {
   ERC1155: 'ERC1155',
 };
 
-export const TX_TYPES = {
```

Comment:
Can you rename this file `nf3.mjs`? Use of `sdk.mjs` is confusing as there is no sdk in this repo.

Reply:
renamed